includes/class-frontend-form-post.php

```diff
@@ -967,7 +967,11 @@ public function send_mail_for_guest( $charging_enabled, $post_id, $form_id, $is_
             }
         }
 
-        if ( isset( $this->form_settings['enable_pay_per_post'] ) && 'true' === $this->form_settings['enable_pay_per_post'] ) {
+        if ( 
+            isset( $this->form_settings['enable_pay_per_post'] )
+                && 'true' === $this->form_settings['enable_pay_per_post'] 
```

Comment:
@hasinur1997 `&&` should be under `isset`, fix the indentation here first. Then check boolean value with `wpuf_validate_boolean` function, use `wpuf_validate_boolean( $this->form_settings['enable_pay_per_post']  )`